### PR TITLE
Update render hook documentation

### PR DESCRIPTION
**Pull request by jmooring**
- [x] Document table render hooks
- [x] Document change to data type returned by render hook Text methods 

<https://deploy-preview-2689--gohugoio.netlify.app/render-hooks/tables/>

Please don't squash commits. Merge upon release of v0.134.0.

### CODE REVIEW

File: content/en/render-hooks/tables.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+## Table cells
+
+The `THead` and `TBody` methods return a slice of table cells, where each element has the following fields:
+
+###### Alignment
+(`string`) The alignment of the text within the table cell, one of `left`, `center`, or `right`.
+
+###### Text
+(`string`) The text within the table cell.
```
Comment (bep) [inline content/en/render-hooks/tables.md:66]: Text = htext.RenderedString = template.HTML. I boldly changed this as part of the table PR and is used for all `Text`. If you think this is a bad idea, please shout.

image

File: content/en/render-hooks/tables.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    {{- if $v }}
+      {{- printf " %s=%q" $k $v | safeHTMLAttr }}
+    {{- end }}
+  {{- end }}>
+  <thead>
+    {{- range .THead }}
+      <tr>
+        {{- range . }}
+          <th {{ printf "style=%q" (printf "text-align: %s" .Alignment) | safeHTMLAttr }}>
+            {{- .Text | safeHTML -}}
```
Comment (bep) [inline content/en/render-hooks/tables.md:86]: `safeHTML` is not needed.

Comment (bep) [reply content/en/render-hooks/tables.md]: No, you are right, this was a mistake. Changing the base type of `hstring.RenderedString` to `template.HTML` does not magically change its type. I'll revert that. Ignore my comments about this ...

File: content/en/render-hooks/blockquotes.md [hunk 3]
```diff
@@ -54,7 +54,7 @@ block = true
 (`string`) The position of the blockquote within the page content.
 
 ###### Text
-(`string`) The blockquote text, excluding the alert designator if present. See the [alerts](#alerts) section below.
+(`hstring.RenderedHTML`) The blockquote text, excluding the alert designator if present. See the [alerts](#alerts) section below.
```
Comment (bep) [inline content/en/render-hooks/blockquotes.md:57]: Note that the motivation behind this type is to have a way to detect circular render loops, which was a very common and very hard to debug user error.

I'm not totally sure the users need to bother with these technical details:

* In all practical situations it behaves like a `string`
* It renders as `template.HTML` which I guess is the most useful information.

File: content/en/render-hooks/images.md [hunk 4]
```diff
@@ -73,11 +73,11 @@ block = true
 
 ###### Text
 
-(`string`) The image description.
+(`hstring.RenderedHTML`) The image description.
 
 ###### Title
 
-(`string`) The image title.
+(`string`) The image titsle.
```
Comment (bep) [inline content/en/render-hooks/images.md:80]: title

**Review by coliff — APPROVED** [review]

**Pull request merged**